lnwallet/channel.go

```diff
@@ -2218,7 +2289,7 @@ func htlcIsDust(incoming, ourCommit bool,
 	// we'll be using an HTLC success transaction as they're the receiver
 	// of this HTLC.
 	case !incoming && !ourCommit:
-		htlcFee = htlcSuccessFee(feePerKw)
+		htlcFee = htlcTimeoutFee(feePerKw)
```

Comment:
correct?

Reply:
same as above

Reply:
When rebased, this section should disappear.